src/uv-common.c

```diff
@@ -155,6 +155,17 @@ static const char* uv__unknown_err_code(int err) {
   return copy != NULL ? copy : "Unknown system error";
 }
 
+#define UV_ERR_NAME_GEN_R(name, _) case UV_ ## name: { snprintf(buf, #name, buflen); break; }
```

Comment:
Style: long line (max length of the line should be 80 characters).
Also I think it should be `snprintf(buf, buflen, #name);`